tasks/eslint.js

```diff
@@ -4,6 +4,10 @@ const CLIEngine = require('eslint').CLIEngine;
 const ClientKitTask = require('clientkit-task');
 
 class EslintTask extends ClientKitTask {
+  constructor(name, config, runner) {
+    super(name, config, runner);
```

Comment:
Can we just set a property for the without needing a constructor?  Worst case a description is a function